app/models/concerns/searchable.rb

```diff
@@ -0,0 +1,17 @@
+module Searchable
+  def index_to_elasticsearch
```

Comment:
I love the `_inline` portion of the following method, perhaps consider putting an `_async` on this method name (unless async is the default and you want it being called in the vast majority of cases)

(Also I realize this is just moving an existing method, so feel free to ignore this suggestion!)

Reply:
The default should always be async so I am going to leave it as-is for now, but if more questions come up about it I'll definitely revisit this! Thanks for reviewing!